Dockerfile

```diff
@@ -12,6 +12,12 @@ WORKDIR /app
 #    apt-get clean && \
 #    rm --recursive --force /var/lib/apt/lists/*
 
+RUN \
+    apt-get update && \
+    apt-get install --assume-yes --no-install-recommends libffi-dev && \
```

Comment:
Why this way, and not via requirements?

Reply:
This is not a Python package. It is related to this error:
https://travis-ci.org/openoereb/pyramid_oereb/jobs/598100771#L1284

Reply:
But it seems this error has nothing to do with your changes? I saw this error also in PR#918, and I also had an error with libffi when building master locally, independently of these PRs. But they seem to be random. When I simply restarted the Travis Job on #918, the error disappeared, and locally, when I clean up some docker images, the error also disappears.

I think it would be better to do a separate issue & PR for this libffi problem, so that your PR contains only the changes related to the functional change. I assume your PR will pass Travis independently if this (if necessary, after 1-2 restarts).

Reply:
Okay, I will undo this change and try it again. :+1: